docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,336 @@
+
+  * Track the progress of a significant implementation.
+
+We could use the blueprints header “status” line to track the progress of some
+features. We could even have a page parsing and listing all the blueprints with
+the title, author, status, and target milestone for that feature.
+
+  * Find the middle ground between “overthinking” and “auto-pilot.”
+
+Last but not least: we are *not trying* to overthink here and/or slow down our
+development processes. The idea is to have a lightweight document, very
+objective that will save us time in the medium and long run. We don’t have to
+overthink by trying to handle any possible scenario outside of ones we actually
+have a use case for. But we should also avoid the “auto-pilot” mode in our
+contributions where we fixing issues as quick as possible without thinking on
+the big picture, it is not healthy for the project.
+
+
+Specification
+*************
+
+One blueprint per topic
+-----------------------
+
+Try to follow the minimalist approach and be concise with content relevant to
+one particular topic. If you have a more general topic to discuss, you should
+set the type as “Epic Blueprint” (more below) but still try to be concise and
+focused on the subject.
+
+File format and location
+------------------------
+
+Our current documentation already uses ReStructuredText (.rst) format, so we
+will adopt .rst format here too. All blueprints will be located inside
+`docs/source/blueprints` with the filename `BPXXX.rst`, where XXX is the number
+of the blueprint. Just pick the next number available for your blueprint.
+
+Write for your audience
+-----------------------
+
+As mentioned before, your blueprint will be read by your peers, future members,
+and future yourself. Keep in mind that your audience is developers with a
+minimal understanding of the Avocado internals, and be kind providing any
+necessary context to understand the problem.
+
+Blueprints types
+----------------
+
+Currently, we have the following blueprint types:
+
+ * Architectural Blueprint: Any blueprint changing or introducing a new core
+   feature or architectural change to Avocado.
+
+ * Process Blueprint: Any blueprint that is not implementing a new core
+   feature, but changing how the project works. This could be, for instance,
+   related to the repositories or processes.
+
+ * Meta Blueprint: A blueprint about blueprints. Like this one and any future
+   blueprint that changes our blueprint's styles and methods.
+
+ * Epic Blueprint: A blueprint that is touching on multiple areas and is too
+   big to have all the documentation in one single blueprint. We could split
+   epic blueprints into smaller blueprints or issues (if they are small and
+   easy to understand). Epic Blueprints are not a merge of all sub-blueprints.
+   Like an epic issue, epic blueprints don't need to detail "how" since is
+   describing a complex problem with many small sub-problems.
```

Comment:
I didn't get the last sentence: Like epic issues, epic blueprints don't need to provide details that the sub-blueprints would clarify?

Reply:
Changing to this:

> Like an epic issue, epic blueprints don't need to detail "how" (or provide details) that the sub-blueprints could have.

Reply:
I guess in this case it should be: "Like an epic issue, epic blueprints don't need to detail the "how" that the sub-blueprints could have.", i.e. just an extra "the" so that it is understandable we refer to the "how" above.